src/Ilios/MeSH/Model/Concept.php

```diff
@@ -117,4 +107,14 @@ public function addTerm(Term $term): void
     {
         $this->terms[] = $term;
     }
+
+    public function getRegistryNumbers(): array
+    {
+        return $this->registryNumbers;
+    }
+
+    public function addRegistryNumber(string $registryNumber): void
+    {
+        $this->registryNumbers[] = $registryNumber;
```

Comment:
I somehow forgot you can add elements this way. I've always used `array_push()`.

Reply:
"syntactical sugar". i like shorthand notation.